app/shared/models/errors/CommonMtdErrors.scala

```diff
@@ -104,7 +104,7 @@ object RuleIncorrectOrEmptyBodyError
 
 object RuleTaxYearRangeInvalidError extends MtdError("RULE_TAX_YEAR_RANGE_INVALID", "A tax year range of one year is required", BAD_REQUEST)
 
-object RuleTaxYearNotEndedError extends MtdError("RULE_TAX_YEAR_NOT_ENDED", "The specified tax year has not yet ended", BAD_REQUEST)
+object RuleTaxYearNotEndedError extends MtdError("RULE_TAX_YEAR_NOT_ENDED", "Tax year not ended", BAD_REQUEST)
```

Comment:
Think this was edited this due to the tech spec? Since this is in shared, might be better to update the tech spec in this case